src/main/drivers/accgyro/accgyro_mpu.c

```diff
@@ -107,21 +110,60 @@ static void mpu6050FindRevision(gyroDev_t *gyro)
  * Gyro interrupt service routine
  */
 #ifdef USE_GYRO_EXTI
+#ifdef USE_SPI_GYRO
+// Called in ISR context
+// Gyro read has just completed
+busStatus_e mpuIntcallback(uint32_t arg)
+{
+    volatile gyroDev_t *gyro = (gyroDev_t *)arg;
+    int32_t gyroDmaDuration = (int32_t)getCycleCounter() - gyro->gyroLastEXTI;
```

Comment:
This has casting on the result of `getCycleCounter()` but this PR also, incorrectly, changes `getCycleCounter` to return an `int32_t` instead of a `uint32_t`.